iocore/net/QUICPacketHandler.cc

```diff
@@ -174,20 +179,16 @@ QUICPacketHandlerIn::_recv_packet(int event, UDPPacket *udp_packet)
     vc->options.ip_proto  = NetVCOptions::USE_UDP;
     vc->options.ip_family = udp_packet->from.sa.sa_family;
 
-    this->action_->continuation->handleEvent(NET_EVENT_ACCEPT, vc);
     qc = vc;
-  }
-
-  if (qc->is_closed()) {
-    this->_ctable.erase(qc->connection_id(), qc);
-    // FIXME QUICNetVConnection is NOT freed to prevent crashes. #2674
-    // QUICNetVConnections are going to be freed by QUICNetHandler
-    // vc->free(vc->thread);
   } else {
-    qc->handle_received_packet(udp_packet);
-    // FIXME This cast is temporal. It'll be removed when we introduce QUICNetHandler.
-    eventProcessor.schedule_imm(static_cast<QUICNetVConnection *>(qc), ET_CALL, QUIC_EVENT_PACKET_READ_READY, nullptr);
+    vc  = static_cast<QUICNetVConnection *>(qc);
+    eth = vc->thread;
   }
+
+  // Push the packet into QUICPollCont
+  udp_packet->data.ptr = vc;
```

Comment:
Hmm, I'm not so thrilled with having this `data` member. Do we have other options?